### PR TITLE
refactor(SpinnakerHttpException): fixed the test cases failing due to change in SpinnakerHttpException in kork-retrofit module

### PR DESCRIPTION
**Pull request by Luthan95**
Fixed the test cases failure due to refactor of SpinnakerHttpException class in kork-retrofit module

### CODE REVIEW

File: rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java [hunk 1]
```diff
@@ -103,20 +105,16 @@ void testSpinnakerHttpExceptionWith404Response() throws Exception {
...
-    // Response is a final class, so straightforward mocking fails.
     String url = "https://some-url";
     Response response =
         new Response(
             url,
             status,
             "arbitrary reason",
             List.of(),
-            new TypedString("{ message: \"unused message due to above mock\" }"));
+            new TypedString("{ \"message\": \"unused message due to above mock\" }"));
```
Comment (dbyron-sf) [inline rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java:115]:
```suggestion
            new TypedString("{ \"message\": \"arbitrary message\" }"));
```

File: rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java [hunk 2]
```diff
@@ -103,20 +105,16 @@ void testSpinnakerHttpExceptionWith404Response() throws Exception {
...
 
-    when(spinnakerHttpException.getResponse()).thenReturn(response);
-    return spinnakerHttpException;
+    return new SpinnakerHttpException(
+        RetrofitError.httpError(url, response, new JacksonConverter(), String.class));
```
Comment (dbyron-sf) [inline rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java:118]: How did you choose JacksonConverter?

Comment (Luthan95) [reply rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java:118]: In almost all cases JacksonConverter is used while constructing RestAdaptor in code...

Comment (dbyron-sf) [reply rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java:118]: Except for [this](https://github.com/square/retrofit/blob/parent-1.9.0/retrofit/src/main/java/retrofit/Platform.java#L70-L72), yes?  Whatever the reason, please include it in a comment in the code.

Comment (Luthan95) [reply rosco-web/src/test/java/com/netflix/spinnaker/rosco/controllers/V2BakeryControllerTest.java:118]: I got it, will replace JacksonConverter with GsonConverter, so no need to add JacksonConverter depedency

**Comment by dbyron-sf** [comment]
included in https://github.com/spinnaker/rosco/pull/943

**Pull request closed without merging**